blueprinter.gemspec

```diff
@@ -11,23 +13,14 @@ Gem::Specification.new do |s|
   s.email       = ['blueprinter@googlegroups.com']
   s.homepage    = 'https://github.com/procore-oss/blueprinter'
   s.summary     = 'Simple Fast Declarative Serialization Library'
-  s.description = 'Blueprinter is a JSON Object Presenter for Ruby that takes business objects and breaks them down into simple hashes and serializes them to JSON. It can be used in Rails in place of other serializers (like JBuilder or ActiveModelSerializers). It is designed to be simple, direct, and performant.'
+  s.description = 'Blueprinter is a JSON Object Presenter for Ruby that takes business objects ' \
+                  'and breaks them down into simple hashes and serializes them to JSON. ' \
+                  'It can be used in Rails in place of other serializers (like JBuilder or ActiveModelSerializers). ' \
+                  'It is designed to be simple, direct, and performant.'
   s.license     = 'MIT'
 
   s.files = Dir['{app,config,db,lib}/**/*', 'CHANGELOG.md', 'MIT-LICENSE', 'Rakefile', 'README.md']
 
   s.required_ruby_version = '>= 2.7'
-
-  s.add_development_dependency 'activerecord', '~> 6.1.7'
-  s.add_development_dependency 'ammeter', '~> 1.1.4'
-  s.add_development_dependency 'factory_bot'
-  s.add_development_dependency 'nokogiri', '>= 1.8.2'
-  s.add_development_dependency 'oj', '~> 3.0'
-  s.add_development_dependency 'pry'
-  s.add_development_dependency 'rake'
-  s.add_development_dependency 'rspec', '~> 3.7'
-  s.add_development_dependency 'rspec-rails', '~> 6.0'
-  s.add_development_dependency 'sqlite3', '~> 1.4.2'
-  s.add_development_dependency 'yajl-ruby', '~> 1.4.1'
-  s.add_development_dependency 'yard', '~> 0.9.11'
```

Comment:
I see that we moved the location of gem dependencies from here to the `Gemfile` as part of [this PR](https://github.com/blueprinter-ruby/blueprinter/pull/10). Was that necessary to get everything working? Just trying to understand the intention of the change here!

Reply:
development dependencies are better suited for Gemfiles right? it also helps when you want to use different versions of dev dependencies for different versions of ruby/rails you want to support. It keeps your gemspec clean and there's no material difference in terms of functionality.

Reply:
It does look like there's a pending default [Rubocop rule](https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecdevelopmentdependencies) for ensuring that development dependencies are enumerated in the `Gemfile` instead of the `gemspec`, so I'm happy to move forward with this change.

As @njbbaer mentioned though, `Appraisal` would be a really nice value add here for testing different Ruby versions. We can follow up on that, though.